doc/release-notes.md

```diff
@@ -5,6 +5,7 @@ Bitcoin Unlimited version 0.12.1 is now available from:
 This is a new minor version release, including ........,
 various bugfixes and updated translations.
 
+<<<<<<< HEAD
```

Comment:
looks like merge annotation accidentally go in

Reply:
fixed